### ISSUE TITLE
make an informative splash page

### ISSUE THREAD

**Issue by iamjessklein** [rockawayhelp/aid-badges-website#1]
The page should explain to people what the concept is and then allow them to sign up for the "beta" in some form.

**Comment by iamjessklein** [comment]
I took a stab at working through what I am thinking. I was imagining a one page (possibly parallax) site that takes you through a scenario with the problem - and solution of aid badges. Something like this (apologize - I sketched on index cards):

cc/ @stevekinney @toolness @threeqube 

![aidbadges](https://cloud.githubusercontent.com/assets/535012/10872206/7610dbde-80ca-11e5-91f6-38324ae5de41.png)

**Comment by toolness** [comment]
This looks great! I like how crisp the messaging is.

I assume there would be an email field at the very bottom for the call-to-action?

**Comment by iamjessklein** [comment]
Yeah I think that's the idea. I guess this might be a good moment to figure out what our plan for follow up might be. It might just be good to get a handful of emails and then follow up with an email and/or questionnaire about the taxonomy. Thoughts?

**Comment by devinbalkind** [comment]
I think it's splendid.  The only change I'd make is to add a personal with a laptop to a slide so it can feel like it's addressing volunteer technical communities (VTCs) like Humanitarian Open Street Map, Standby Task Force, etc.  That laptop person could be on the "many people will volunteer - which is great" slide or  "it can be difficult to know what each volunteer is capable of" one.  Aside from that small change, I think you've nailed it with the messaging.

**Comment by ariammogos** [comment]
Jess this is awesome, great messaging. I agree, I think getting a handful of emails and then sorting out taxonomy is a good idea. I can help with gathering email addresses...is there a communication plan for this project? Is the goal to have a group who we'll be continuously testing with?

**Comment by iamjessklein** [comment]
Thanks @devinbalkind and @MogiBear - I will incorporate a laptop into the next iteration.

Re: communications - the idea here is twofold - 1. to get people to sign up so that we can develop a cohort to usertest badges with and 2. to gain insight from experienced aid communities as to what "badges" would be appropriate.

I wrote a new issue for us to think this through in #2  - are either of you willing to take that on?

**Comment by ariammogos** [comment]
I can write the copy, how is this project being managed? Is the Trello still up?

**Comment by iamjessklein** [comment]
ha. organization - I've fallen off the wagon. Why don't we just start by writing github issues? Then we can pull them into waffle.io

**Comment by devinbalkind** [comment]
You can get a trello/kanban style experience with Github issues using
https://waffle.io/

**Comment by iamjessklein** [comment]
Just started to mock up the visuals - here is scene 1. If this is on mobile, just the middle section will be visible. 

![aid-desktop](https://cloud.githubusercontent.com/assets/535012/11023134/094a0aec-863e-11e5-9cd4-9fd178df2bb2.png)

**Comment by toolness** [comment]
That looks amazing.

**Comment by ariammogos** [comment]
Ditto, love love love this. Steve and I spoke today and we are both pumped!

**Comment by iamjessklein** [comment]
Thanks guys! My goal is to get this all drawn up over the next week or so.

**Comment by devinbalkind** [comment]
Wow.  Amazing. 👏

**Comment by iamjessklein** [comment]
Ok - started the second "screen" 

![aidbadges3](https://cloud.githubusercontent.com/assets/535012/11069829/326d7800-87a6-11e5-8e5a-a0c8dbad056d.png)

**Comment by devinbalkind** [comment]
Are those people with rain ponchos carrying signs?

**Comment by iamjessklein** [comment]
Yes they are. The sign is one half a heart and then I thought the volunteers could have signs with the other heart half.

**Comment by devinbalkind** [comment]
Oh my. That's such a beautiful concept. Love it.

**Comment by iamjessklein** [comment]
Something kind of like this https://instagram.com/p/98zD2Yyvzp/

**Comment by ariammogos** [comment]
So awesome Jess.

**Comment by iamjessklein** [comment]
I think when people volunteer they should first raise their hands. I will need to make everyone look unique - but this is the general idea:

![aidbadges3](https://cloud.githubusercontent.com/assets/535012/11107569/3a7005ec-88ad-11e5-9581-2db4c747bccb.png)

**Comment by threeqube** [comment]
@iamjessklein I absolutely love these sketches! 

qq: like you mentioned earlier, are you gonna try one where the volunteers have the other half of a heart?

**Comment by iamjessklein** [comment]
@threeqube thanks, yeah. So I realized that there needs to be a space before the hearts arrive where it's hard to tell who has what skill. I think that I need to have the volunteers sign up with iPads or something before they make the "heart connection".

**Comment by toolness** [comment]
Yes! I **really** like this heart connection narrative. We should totally carry this metaphor through into the product as it evolves!

**Comment by threeqube** [comment]
Ah got it. That makes sense. Excited to see the next storyboard. :+1:

**Comment by iamjessklein** [comment]
So the last frame I was thinking could pan out horizontally - instead of vertically - so it shifts to the side.... left = affected right = volunteers

It's hard to communicate with a still image, but imagine this scene animated: 

<img width="1054" alt="screen shot 2015-12-29 at 7 04 06 pm" src="https://cloud.githubusercontent.com/assets/535012/12044430/1224526e-ae5f-11e5-8e9e-50e178b1fccf.png">

**Comment by iamjessklein** [comment]
---->>> ZOOM IN <<<----

<img width="675" alt="screen shot 2015-12-29 at 7 47 15 pm" src="https://cloud.githubusercontent.com/assets/535012/12044752/060faad6-ae65-11e5-8765-9b91cfe21a07.png">

**Comment by iamjessklein** [comment]
words come up from below and the characters move off screen:

<img width="676" alt="screen shot 2015-12-29 at 7 55 43 pm" src="https://cloud.githubusercontent.com/assets/535012/12044815/2f33eb38-ae66-11e5-9841-06551b342b7c.png">

**Comment by iamjessklein** [comment]
The signs will "drop" down on scroll and become badges.

<img width="991" alt="screen shot 2016-01-23 at 8 33 19 pm" src="https://cloud.githubusercontent.com/assets/535012/12533907/9585cab0-c210-11e5-8803-1d3b7813d687.png">

**Comment by iamjessklein** [comment]
<img width="838" alt="screen shot 2016-01-23 at 9 20 40 pm" src="https://cloud.githubusercontent.com/assets/535012/12534072/3183d028-c217-11e5-8492-ede3776b8b92.png">

**Comment by iamjessklein** [comment]
hearts drop... into final section:
<img width="674" alt="screen shot 2016-01-23 at 9 39 44 pm" src="https://cloud.githubusercontent.com/assets/535012/12534151/e6f7b562-c219-11e5-9791-3143eb162478.png">

**Comment by iamjessklein** [comment]
This is the first transition (more or less):

![comp-1- converted](https://cloud.githubusercontent.com/assets/535012/12537734/6e0e5432-c294-11e5-865d-e0447cfed71b.gif)

**Comment by threeqube** [comment]
These are amazing. I :heart: them all.

**Comment by iamjessklein** [comment]
Thanks @threeqube 

I got a little bit further:

![new](https://cloud.githubusercontent.com/assets/535012/12539598/d2df7190-c2c4-11e5-8c83-358f32e6ecda.gif)

**Comment by iamjessklein** [comment]
For the responsive version - I'm imagining that the screen would look like this ... with the "animation" being turned into a little movie. 

![mobile portrait](https://cloud.githubusercontent.com/assets/535012/12869112/afd38334-cce3-11e5-9ee3-fbb7ec08e156.png)
